core/roboconf-core/src/test/java/net/roboconf/core/utils/ProcessStoreTest.java

```diff
@@ -0,0 +1,39 @@
+package net.roboconf.core.utils;
+
+import java.io.IOException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+
+/**
+ * 
+ * @author Pierre-Yves Gibello - Linagora
+ *
+ */
+public class ProcessStoreTest {
+	
+	String applicationName = "app1";
+	String scopedInstancePath = "/anything";
+
+	@Test
+	public void testProcessFunctions() throws IOException {
+		Assert.assertNull(ProcessStore.getProcess(applicationName, scopedInstancePath));
+		
+		Process process = (new ProcessBuilder("ls")).start();
```

Comment:
What happens on another OS?
What happens if you put "toto says hi" instead? If it does not work with "toto", then you should verify the test can run on the system. Junit's **Assume** aims at it.